File: isegoria/src/main/java/com/eulersbridge/isegoria/profile/settings/AboutActivity.java
package com.eulersbridge.isegoria.profile.settings;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

import com.eulersbridge.isegoria.R;

public class AboutActivity extends AppCompatActivity  {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.about_activity);

        findViewById(R.id.back_button).setOnClickListener(view -> onBackPressed());
    }
}

File: isegoria/src/main/kotlin/com/eulersbridge/isegoria/profile/settings/AboutActivity.kt
package com.eulersbridge.isegoria.profile.settings

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

import com.eulersbridge.isegoria.R
import kotlinx.android.synthetic.main.activity_about.*

class AboutActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_about)

        backButton.setOnClickListener({ onBackPressed() })
    }
}
